swarm/tools/runs_gc.py

```
@@ -71,16 +72,20 @@ def age_days(self) -> float:
-                try:
-                    total += entry.stat().st_size
-                except OSError:
-                    pass
+        # Bolt: Replaced pathlib.Path.rglob with os.scandir for faster recursive traversal and fewer stat() calls
+        with os.scandir(path) as it:
+            for entry in it:
+                if entry.is_file():
+                    try:
+                        total += entry.stat().st_size
+                    except OSError:
+                        pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)